docs/secure-connections/sauce-connect/installation.md

```diff
@@ -70,6 +66,19 @@ tar xvf ./sc-4.7.1-linux.tar.gz
 export PATH="$HOME/sc-4.7.1-linux/bin:$PATH"
 ```
 
+
+## Using Docker Containers to Install/Run Sauce Connect
```

Comment:
This section would require "What You’ll Need" 
- Installed and configured [docker](https://www.docker.com/)